crates/polars-plan/dsl-schema.json

```diff
@@ -0,0 +1,10762 @@
+{
```

Comment:
Given that the file is so huge and probably will change often, maybe we should move to a hash of the schema?

Reply:
I used the whole file, so the PR authors and reviewers can easily see in the PR diff how the schema changed, if it was intentional, and can then use that info to help decide if major or minor needs to be incremented.

I could change it to a hash if we are ok with just knowing that the DSL changed, and let the author/reviewers to figure the rest out for themselves.

As an alternative I could generate the schema from the target branch and PR branch on the CI and post the diff as a  comment, but that would require building the target branch (main) again every time the CI runs for a PR.

Reply:
I think the hash is fine. I think it will save a lot of git bloat. They can generate the schema manual from main if needed.